src/renderers/dom/__tests__/ReactDOMProduction-test.js

```diff
@@ -178,6 +178,9 @@ describe('ReactDOMProduction', () => {
     expect(function() {
       class Component extends React.Component {
         render() {
+          // FIXME: undefined is a valid return type in Fiber, at least in the
+          // current implementation. It's treated as empty. I don't know why we
+          // sometimes get false positives.
```

Comment:
>this sometimes gets a false positive when I run the record-tests script with the ReactDOMFeatureFlag turned on locally

Yes, I'm sure it's the reason we've been getting false positives. (Had it many times and pinpointed to the same cause.)

We do need it to fix it. It should be failing in Fiber so I'm surprised it [doesn't](https://github.com/facebook/react/blob/c74034589e6875402b004b243a3debeda406a7ae/scripts/fiber/tests-passing.txt#L491)..